test/Microsoft.Health.Fhir.Tests.E2E/Rest/HistoryTests.cs

```diff
@@ -213,28 +212,15 @@ public async Task WhenGettingSystemHistory_GivenAQueryThatReturnsMoreThan10Resul
             newResources.Add(await Client.CreateAsync(Samples.GetJsonSample("ObservationWith1MinuteApgarScore").ToPoco()));
             newResources.Add(await Client.CreateAsync(Samples.GetJsonSample("ObservationWith20MinuteApgarScore").ToPoco()));
 
-            Thread.Sleep(1000); // Leave a small gap in the timestamp
-            var before = DateTime.UtcNow;
+            var lastAdded = newResources.Last<Resource>();
+            var before = lastAdded.Meta.LastUpdated.Value.AddMilliseconds(500);
```

Comment:
Why +500ms instead of just lastAdded.Meta.LastUpdated?

Reply:
_before is not inclusive, it is exclusive.  In theory it should fail to return the change if the timestamp is directly equal.

Reply:
Got it, maybe its worth adding that comment in the test?